docs/source/getting_started/python-guide.rst

```diff
@@ -0,0 +1,81 @@
+Sending transactions with Python library
+========================================
+
+Prerequisites
+-------------
+
+.. note:: the library only works in Python 3 environment (Python 2 is not supported now).
+
+To use Iroha Python library, you need to get it from the `repository <https://github.com/hyperledger/iroha-python>`_ or via pip3: 
+
+.. code-block:: shell
+
+	pip3 install iroha
+
+Now, as we have the library, we can start sending the actual transactions. 
+
+Running example transactions
+----------------------------
+
+If you only want to try what Iroha transactions would look like, you can simply go to the examples from the repository `here <https://github.com/hyperledger/iroha-python/tree/master/examples>`_. Let's check out the `tx-example.py` file. 
+
+Here are Iroha dependancies. Python library generally consists of 3 parts: Iroha, IrohaCrypto and IrohaGrpc which we need to import: 
+
+.. code-block:: shell
+	from iroha.primitive_pb2 import can_set_my_account_detail
```

Comment:
I suggest to put in a separate example, after the note, for example

Reply:
I do not understand, could you please clarify?